src/TVDB/Web/WebInterface.cs

```diff
@@ -593,7 +602,7 @@ public WebInterface(string apiKey)
 			{
 				foreach (ZipArchiveEntry entry in archive.Entries)
 				{
-					entry.ExtractToFile(Path.Combine(dirInfo.Name, entry.Name), true);
+					entry.ExtractToFile(Path.Combine(dirInfo.FullName, entry.Name), true);
```

Comment:
Using the FullName property could be a bit nasty because it will create all containing sub folders of the zip file and we don't know if the developer of TheTVDB API will change the packing behavior. My intention was to create a flat folder which contains all data regardless of the original struct of the zip file.